src/components/DatePicker/DatePicker.mdx

```diff
@@ -1,70 +1,15 @@
 import {
-  ArgsTable,
+  ArgTypes,
   Canvas,
+  Controls,
```

Comment:
This is a question I have: What are we using the Controls import for? I removed it from the ones I did and nothing changed. But maybe I am missing it?

Reply:
You're right! I missed that and it should be removed. `ArgTypes` is the correct component to render the prop table and Controls doesn't do anything for us right now.